modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java

```diff
@@ -177,11 +179,11 @@ public void tearDown() throws Exception {
 	public void testExportCompanyScopeObjectEntry() throws Exception {
 		_objectDefinition1 = _publishObjectDefinition(
 			TestPropsValues.getCompanyId(), "TestObject",
-			TestPropsValues.getUser());
```

Comment:
Maybe not right now... but we should move these tests to objects modules. We should not be mixing the framework with the client code.

Reply:
I agree, with those changes are not only applied to the tests. I think we need to create a taks to refactor the current code, as we are making some development also based on Objects

Reply:
ok

Reply:
Can you create the tasks?

Reply:
https://liferay.atlassian.net/browse/LPS-200224 here we go